src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -282,37 +282,33 @@
            :gradient-cover?          true
            :customization-color      (:color account)}
           [rn/view
-           [rn/pressable
-            {:on-press #(rf/dispatch [:navigate-to-within-stack
-                                      [:screen/wallet.transaction-details
-                                       :screen/wallet.transaction-confirmation]])}
```

Comment:
@vkjr I believe this `on-press` is not needed as there is no screen transition required from the confirmation page, and we do not have a `wallet.transaction-details` screen. This is introduced in https://github.com/status-im/status-mobile/pull/21657. Let me know if I missed anything.

Reply:
@smohamedjavid, wow, thanks for noticing! That is a leftover that I made for developer convenience while worked on mentioned PR but forgot to remove 🤦